app/controllers/reorder_sub_sections_controller.rb

```diff
@@ -36,29 +46,14 @@ def submitted_positions
 
   def set_positions(positions)
     positions.each do |sub_section|
-      SubSection.find(sub_section["id"]).update(position: sub_section["position"])
+      SubSection.find(sub_section["id"]).update_column(:position, sub_section["position"])
```

Comment:
I feel like this should be:

```
        sub_section = coronavirus_page.sub_sections.find(sub_section_data["id"])
        sub_section.update_column(:position, sub_section_data["position"])
```

for consistency with announcements and to fix a small oddity that a submission of this form could edit SubSection objects that have don't have a relationship with the current coronavirus page.